tests/integration_tests/lightgbm_tuner_tests/test_alias.py

```diff
@@ -37,40 +41,56 @@ def test_handling_alias_parameter() -> None:
 
 
-def test_handling_alias_metrics() -> None:
-
-    for alias in [
-        "lambdarank",
-        "rank_xendcg",
-        "xendcg",
-        "xe_ndcg",
-        "xe_ndcg_mart",
-        "xendcg_mart",
-        "ndcg",
-    ]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "ndcg"
-
-    for alias in ["mean_average_precision", "map"]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "map"
-
-    for alias in ["regression", "regression_l2", "l2", "mean_squared_error", "mse"]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "l2"
-
-    lgbm_params = {}
-    _handling_alias_metrics(lgbm_params)
-    assert lgbm_params == {}
-
-    lgbm_params = {"metric": "auc"}
-    _handling_alias_metrics(lgbm_params)
-    assert lgbm_params["metric"] == "auc"
-
-    for alias in ["l2_root", "root_mean_squared_error", "rmse"]:
-        lgbm_params = {"metric": alias}
+@pytest.mark.parametrize(
+    "aliases, expect",
+    [
+        (
+            [
+                "ndcg",
+                "lambdarank",
+                "rank_xendcg",
+                "xendcg",
+                "xe_ndcg",
+                "xe_ndcg_mart",
+                "xendcg_mart",
+            ],
+            "ndcg",
+        ),
+        (["mean_average_precision", "map"], "map"),
+        (["rmse", "l2_root", "root_mean_squared_error"], "rmse"),
+        (["l1", "regression_l1", "mean_absolute_error", "mae"], "l1"),
+        (["l2", "regression", "regression_l2", "mean_squared_error", "mse"], "l2"),
+        (["auc"], "auc"),
+        (["binary_logloss", "binary"], "binary_logloss"),
+        (
+            [
+                "multi_logloss",
+                "multiclass",
+                "softmax",
+                "multiclassova",
+                "multiclass_ova",
+                "ova",
+                "ovr",
+            ],
+            "multi_logloss",
+        ),
+        (["cross_entropy", "xentropy"], "cross_entropy"),
+        (["cross_entropy_lambda", "xentlambda"], "cross_entropy_lambda"),
+        (["kullback_leibler", "kldiv"], "kullback_leibler"),
+        (["mape", "mean_absolute_percentage_error"], "mape"),
+        (["auc_mu"], "auc_mu"),
+        (["custom", "none", "null", "na"], "custom"),
+        ([], None),  # If "metric" not in lgbm_params.keys(): return None.
+    ],
+)
+def test_handling_alias_metrics(aliases: List[str], expect: str) -> None:
+    if len(aliases) > 0:
+        for alias in aliases:
+            lgbm_params = {"metric": alias}
+            _handling_alias_metrics(lgbm_params)
+            assert lgbm_params["metric"] == expect
+    else:
+        lgbm_params = {}
         _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "rmse"
+        print("null case")
```

Comment:
A nit, but could you remove this line? The message is short and it looks like code for debugging.

Reply:
Yes, thank you for pointing out.